es6/server/db/index.js

```diff
@@ -1,2 +1,3 @@
+require('dotenv').config();
 const pgp = require('pg-promise')();
 export const db = pgp('postgresql://CzA:King301094@localhost:5432/stackoverflow');
```

Comment:
you should have this `postgresql://CzA:King301094@localhost:5432/stackoverflow` as an environment variable because it is a sensitive data.

Reply:
noted, thanks